Sources/ComposableArchitectureMacros/ViewActionMacro.swift

```diff
@@ -0,0 +1,158 @@
+@preconcurrency import SwiftDiagnostics
+import SwiftSyntax
+import SwiftSyntaxMacroExpansion
+import SwiftSyntaxMacros
+
+public struct ViewActionMacro: MemberMacro {
+  public static func expansion<Declaration, Context>(
+    of node: SwiftSyntax.AttributeSyntax,
+    providingMembersOf declaration: Declaration,
+    in context: Context
+  ) throws -> [SwiftSyntax.DeclSyntax]
+  where Declaration: SwiftSyntax.DeclGroupSyntax, Context: SwiftSyntaxMacros.MacroExpansionContext {
+    guard declaration.hasStoreVariable
+    else {
+      // TODO: Fix it to add `let store: StoreOf<<#Feature#>>`?
+      context.diagnose(
```

Comment:
I don't think this fix-it is actually possible since we need a node insertion rather than replacement. But maybe you know some fancy tricks!

Reply:
Oh, well I guess we could replace the entire `declaration` node with a new version of itself that has a member added...